.travis.yml

```diff
@@ -7,7 +7,7 @@ notifications:
 # Run latest 3.x and 2.x next, to get quick compatibility results.
 # Then run the remainder.
 python:
-  - "pypy"
+  - "pypy-5.3.1"
```

Comment:
Since we switch to Trusty, the default version of pypy in Tursty is PyPy 5.4.1. There is a well-known error in it : `TypeError: PyPy does not yet implement the new buffer interface`  see this bug #2163.